functional/start_etcd

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
```

Comment:
For those who want to run tests in qemu (and any other coreos environments) without specified cloud-config.